rafs/Cargo.toml

```diff
@@ -3,10 +3,12 @@ name = "rafs"
 version = "0.1.0"
 authors = ["The Nydus Developers"]
 edition = "2018"
+license = "Apache-2.0 OR BSD-3-Clause"
 
 [dependencies]
 anyhow = "1.0.35"
-arc-swap = "0.4.6"
+# pin arc-swap because 1.x version of ArcSwapAny does not implement Clone
+arc-swap = "=0.4"
```

Comment:
But this patch still implements `Clone` itself?

Reply:
Ah, my bad. Should have dropped the Clone implementation. Fixed.